README.md

```diff
@@ -173,7 +173,7 @@ poetry install
 Then you can run the script with:
 
 ```
-poetry run idasen_controller/main.py <command>
+poetry run python -m idasen_controller.main <command>
```

Comment:
> and I have fixed the README

Isn't the `-m` required here? https://github.com/rhyst/linak-controller/blob/master/README.md?plain=1#L178